libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/gutenberg/GutenbergPropsBuilder.kt

```diff
@@ -29,7 +29,7 @@ data class GutenbergPropsBuilder(
             enableXPosts = enableXPosts,
             enableUnsupportedBlockEditor = enableUnsupportedBlockEditor,
             canEnableUnsupportedBlockEditor = unsupportedBlockEditorSwitch,
-            enableAudioBlock = enableAudioBlock,
+            isAudioBlockMediaUploadEnabled = isAudioBlockMediaUploadEnabled,
```

Comment:
These changes come from this [PR](https://github.com/wordpress-mobile/WordPress-Android/pull/14680), that it was required to merge it here due to conflicts in the Gutenberg reference.